prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.ts

```diff
@@ -61,6 +67,15 @@ export class EnrolleeOverviewComponent extends AdjudicationContainerComponent im
       .subscribe(params => this.loadEnrollee(params.id));
 
     this.action.subscribe(() => this.loadEnrollee(this.route.snapshot.params.id));
+
+    this.paperEnrolmentResource.getEnrolleeById(+this.route.snapshot.params.id)
+      .subscribe((enrollee: HttpEnrollee) => this.enrollee = enrollee);
+
+    this.paperEnrolmentResource.getAdjudicationDocuments(+this.route.snapshot.params.id)
+      .subscribe((documents: EnrolleeAdjudicationDocument[]) => {
+        this.documents = documents
+      });
+    var x = 2;
```

Comment:
unused variable

Reply:
Ops! removed, great catch!

Reply:
Also, never use `var` start with `const` and if needed then use `let`